README.md

```diff
@@ -66,14 +68,115 @@ Type: t2.micro
 AutoScaling: (none)
 ```
 
+### S3
+
+In the bucket context, calculates the objects inside the bucket and alerts if the size reaches the threshold.
```

Comment:
calculates the size of the objects, right?

Reply:
Yes, each object inside the bucket will be [summarized](https://github.com/NETWAYS/check_cloud_aws/blob/feature/cloudfront/internal/s3/bucket.go#L29)

Reply:
should be made clear in the description